xml/System.Security.Cryptography/SHA1Managed.xml

```diff
@@ -219,21 +218,21 @@ This method is called by the public <xref:System.Security.Cryptography.HashAlgor
         <ReturnType>System.Void</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="array" Type="System.Byte[]" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netstandard-2.0;xamarinandroid-7.1" />
+        <Parameter Name="array" Type="System.Byte[]" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1" />
```

Comment:
@joelmartinez here's my attempt to fix the signature problem. I also think that the MemberSignature  entries is missing an entry for the signature for xamarinios-10.8